Production/govuk_ios/Resources/Assets.xcassets/OnboardingImages/onboarding_placeholder_screen_1.imageset/Contents.json

```diff
@@ -1,9 +1,8 @@
 {
   "images" : [
     {
-      "filename" : "onboarding_placeholder_screen_1.png",
-      "idiom" : "universal",
-      "scale" : "1x"
+      "filename" : "onboarding_placeholder_screen_1.svg",
```

Comment:
I have given the new images the same filename of the image they are replacing.

Unsure if I should remove the term `placeholder` from the filenames and image reference names in the app but that would require a code change also and this change was just for replacing images?